<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>793</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>8</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Log Parser V5</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/log_oxigen_32.png</normaloff>:/icons/log_oxigen_32.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="Canada"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget_General">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Parameters">
       <attribute name="title">
        <string>Parsing</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>Path:   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_Path"/>
          </item>
          <item>
           <widget class="QToolButton" name="btn_SelectDir">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="horizontalSpacing">
           <number>-1</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QDateEdit" name="date_From">
            <property name="displayFormat">
             <string>yyyy/MM/dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_3">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string> To:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string> Performance Trigger:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string> From:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>sec </string>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QSpinBox" name="spin_contextLOL">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string> Context:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QDateEdit" name="date_To">
            <property name="displayFormat">
             <string>yyyy/MM/dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
            <property name="date">
             <date>
              <year>2100</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QDoubleSpinBox" name="spin_perfTrigger">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.500000000000000</double>
            </property>
            <property name="maximum">
             <double>999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>3.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="10">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>previous lines</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Output: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edt_output"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="edt_console">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="tabStopWidth">
           <number>80</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Config">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_AppLogLevels">
              <property name="text">
               <string>Log Levels:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_LogLevels">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>80</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="showDropIndicator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::IgnoreAction</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="flow">
               <enum>QListView::TopToBottom</enum>
              </property>
              <property name="isWrapping" stdset="0">
               <bool>true</bool>
              </property>
              <property name="resizeMode">
               <enum>QListView::Fixed</enum>
              </property>
              <property name="layoutMode">
               <enum>QListView::Batched</enum>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="viewMode">
               <enum>QListView::ListMode</enum>
              </property>
              <property name="modelColumn">
               <number>0</number>
              </property>
              <property name="selectionRectVisible">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>LOG</string>
               </property>
               <property name="checkState">
                <enum>Checked</enum>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>FATAL</string>
               </property>
               <property name="checkState">
                <enum>Checked</enum>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EXCEPTION_TRACK</string>
               </property>
               <property name="checkState">
                <enum>Checked</enum>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>LEAK</string>
               </property>
               <property name="checkState">
                <enum>Checked</enum>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>WARNING</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>STATISTICS</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_Exclusions">
            <property name="text">
             <string>Exclusions:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="edt_Exclusions"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_Categories">
            <property name="text">
             <string>Categories:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="edt_Categories"/>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
       <zorder></zorder>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_statusbar">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_files">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Files processed: </string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_processed_files">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>/</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_total_files">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_lines">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Lines processed: </string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_processed_lines">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>/</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_total_lines">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_log_entries">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_log_entries_count">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string> log entries to analyze</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_elapsed_time">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Elapsed Time: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_elapsed_time">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_progress_bar">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSave_config_to_file"/>
    <addaction name="actionLoad_config_from_file"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionParse"/>
    <addaction name="actionClean_logs"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_Folder"/>
    <addaction name="actionRecursive"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Folder"/>
   <addaction name="separator"/>
   <addaction name="actionParse"/>
   <addaction name="actionStop"/>
   <addaction name="actionClean_logs"/>
   <addaction name="separator"/>
   <addaction name="actionSave_config_to_file"/>
   <addaction name="actionLoad_config_from_file"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/shutdown_32.png</normaloff>:/icons/shutdown_32.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionSave_config_to_file">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Arrow-down-icon_32.png</normaloff>:/icons/Arrow-down-icon_32.png</iconset>
   </property>
   <property name="text">
    <string>Save options to file...</string>
   </property>
  </action>
  <action name="actionLoad_config_from_file">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Arrow-up-icon_32.png</normaloff>:/icons/Arrow-up-icon_32.png</iconset>
   </property>
   <property name="text">
    <string>Load options from file...</string>
   </property>
  </action>
  <action name="actionParse">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Categories-preferences-system-icon_32.png</normaloff>:/icons/Categories-preferences-system-icon_32.png</iconset>
   </property>
   <property name="text">
    <string>Parse</string>
   </property>
   <property name="iconText">
    <string>Parse</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionClean_logs">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/bin-red-full-icon_32.png</normaloff>:/icons/bin-red-full-icon_32.png</iconset>
   </property>
   <property name="text">
    <string>Clean logs</string>
   </property>
   <property name="toolTip">
    <string>Clean log files</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionSelect_Folder">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/folder_search_32.png</normaloff>:/icons/folder_search_32.png</iconset>
   </property>
   <property name="text">
    <string>Select Folder...</string>
   </property>
  </action>
  <action name="actionRecursive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Recursive</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/stop_32.png</normaloff>:/icons/stop_32.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop log file parsing</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
